src/Microsoft.Framework.Runtime.Interfaces/IServiceProviderLocator.cs

```diff
@@ -5,8 +5,14 @@
 
 namespace Microsoft.Framework.Runtime.Infrastructure
 {
+    /// <summary>
+    /// The <see cref="IServiceProviderLocator"/> provides access to the current <see cref="IServiceProvider"/>.
```

Comment:
Putting the interface itself as a cref isn't really necessary here. In fact, don't mention it at all

> Provides access to the current ...